en-US/Free-Range_Monsters/themes/pdf-theme.yml

```diff
@@ -0,0 +1,46 @@
+extends:
+  - default
+base:
+  font-size: 12
+page:
+  size: Letter
+title-page:
```

Comment:
Here's an example of what the title page can look like with these settings:
<img width="687" height="887" alt="image" src="https://github.com/user-attachments/assets/719f1fbc-e386-4732-8a1a-1b7d70ba7c45" />